Mistica/Source/Components/Feedback/FeedbackView.swift

```diff
@@ -54,24 +54,58 @@ public class FeedbackView: UIView {
         return animation
     }()
 
+    var iconAccessibilityIdentifier: String? {
+        get {
+            icon.accessibilityIdentifier ?? animatedIcon.accessibilityIdentifier
```

Comment:
since this sets a value in `accessibilityIdentifier` for _icon_ and _animatedIcon_ at the same time, it may only be necessary to return one value because the second one will have the same value.
```suggestion
            icon.accessibilityIdentifier
```